apps/teams/tests/test_creation.py

```diff
@@ -24,12 +24,26 @@ def test_create_for_user(self):
         assert [SUPER_ADMIN_GROUP] == [group.name for group in membership.groups.all()]
 
 
-def test_create_team_view(db, client):
+def test_slug_generation_on_team_creation(db, client):
```

Comment:
I can't seem to find the docs now, but the other day Simon linked me to pytest docs where they recommend that one reserve the `db` fixture for custom fixtures and rather use the `pytest.mark.django_db()` fixture for normal tests